cogs/welcome/welcome.py

```diff
@@ -537,6 +648,112 @@ def check(message: discord.Message):
         await self.config.guild(ctx.guild).get_attr(key).set(greetings)
         return
 
+    # [p]welcomeset greetings image
+    @greetings.group(name="image")
+    async def image(self, ctx: Context):
+        """Base command for the image command group"""
+        pass
+
+    # [p]welcomeset greetings image template
+    @image.command(name="template")
+    async def imageTemplate(self, ctx: Context):
+        await ctx.send(
+            "Here is the welcome image template so you can make your own! For best results please render the image at [IMAGE_DPI]dpi, [IMAGE_DIMENSION_HEIGHT] x [IMAGE_DIMENSION_WIDTH]. The bot will try to make it conform automatically but mileage may vary.",
```

Comment:
Do we substitute `[IMAGE_DPI]`, `[IMAGE_DIMENSION_HEIGHT]` and `[IMAGE_DIMENSION_WIDTH]` with their actual constant values?

Reply:
Yes, I'll remove the brackets to make it more clear